enterprise/templates/enterprise/_enterprise_customer_sidebar.html

```diff
@@ -0,0 +1,7 @@
+<img class="enterprise-logo" src="{{ enterprise_customer.branding_configuration.logo.url }}" alt="{{ enterprise_customer.name }}"/>
```

Comment:
Should we check for the existence of a branding_configuration the way we are doing [here](https://github.com/edx/edx-enterprise/blob/master/enterprise/templates/enterprise/grant_data_sharing_permissions.html#L27)?

Reply:
Yes, Its better to not display the enterprise logo placeholder instead of the actual logo if there is no branding available for the enterprise. Thanks for pointing this out.